src/layers/core/grid-layer/grid-layer-vertex.glsl

```diff
@@ -16,25 +15,12 @@ attribute vec3 instancePickingColors;
 uniform float renderPickingBuffer;
 uniform vec3 selectedPickingColor;
 
-// Projection uniforms
-uniform mat4 worldMatrix;
-uniform mat4 viewMatrix;
-uniform mat4 worldInverseTransposeMatrix;
-
 // Custom uniforms
 uniform float extruded;
 uniform float lonOffset;
 uniform float latOffset;
 uniform float opacity;
 
-// Lighting constants
-const vec3 ambientColor = vec3(0.8, 0.8, 0.8);
-const vec3 pointLocation = vec3(1.5, 1.5, 5.);
-const vec3 pointColor = vec3(0.7, 0.7, 0.7);
-const vec3 pointSpecularColor = vec3(0.6, 0.6, 0.6);
-const float shininess = 3.;
-const float pointLightAmbientCoefficient = 1.;
-
 // A magic number to scale elevation so that 1 unit approximate to 100 meter
 #define ELEVATION_SCALE 80.
```

Comment:
I believe that this is being addressed in @heshan0131 's coming Grid and HexagonLayer update.